src/averageAges.js

```diff
@@ -37,7 +43,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const filteredWomen = people.filter(woman => {
+    return withChildren
+      ? woman.sex === 'f' && people.some(person => person.mother === woman.name)
```

Comment:
It is not necessary to check if the person is female, because their name is written in child's `mother` property

Reply:
yeah, fixed it already, thx)